assets/src/components/selection/event-select/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,40 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EventSelect with 2 events should render and match snapshot 1`] = `
```

Comment:
This is a snapshot created by Jest and the new tests I added.  This should _never_ get modified directly.